aimsAndScope.md

```diff
@@ -1,10 +1,22 @@
 # Aims and scope
 
-*Journal of Cheminformatics* is an open access journal publishing original peer-reviewed research in all
+*Journal of Cheminformatics* is an open science journal publishing original peer-reviewed research in all
 aspects of cheminformatics and molecular modelling.
 
 Coverage includes, but is not limited to:
 
 * chemical information systems, software and databases, and molecular modelling
 * chemical structure representations and their use in structure, substructure, and similarity searching of chemical substance and chemical reaction databases
 * computer and molecular graphics, computer-aided molecular design, expert systems, QSAR, and data mining techniques
+
+## Open Science
+
+The journal has the following open science ambitions:
+
+* publish research or software that is entirely reproducible by third parties
+* access to any datasets, software and algorithms that are required to reach the conclusions
+  stated in the paper mus be accessible without the need for registration, login or agreement
```

Comment:
```suggestion
  stated in the paper must be accessible without the need for registration, login or agreement
```

Reply:
having in mind e.g. GitHub resources are accessible only for registered users, is this what we want?

Reply:
apologies, I am wrong, no need to login to see source code (but still need to login to use e.g. issue tracker, which is outside of the requirements)

Reply:
still, many open resources, API, database access, etc.  do require free registration , API keys, etc. , at least for technical reasons.

Reply:
I think this is captured/relativised in the "any XXX required to reach the conclusions stated in the paper" part ... at least for our CASMI2016 paper we discussed with the editor (@steinbeck iirc) upfront and came up with a compromise solution as we could not require all participants in the contest to have open methods & comply with JCheminf requirements, but we were able to make all the *data* open so the publication conclusions were reproducible and thus compliant. @egonw maybe consider adding a statement that this could be discussed with editors; or alternatively elaborate more under the different paper types?

Reply:
Yes, that is still possible... we could cover that, but this actually has been the exact text in the last two years... I don't want to loosen it too much, because we more often have submissions that are not open enough, than that they are too open...

Reply:
I am fine with the old text, I like it - and I reached out when I was in doubt and got a good, fair answer quickly.